batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDao.java

```diff
@@ -0,0 +1,143 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.constants.ModelConstants;
+import com.lftechnology.batch7crud.db.DBConnection;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by Grishma Shrestha grishmashrestha@lftechnology.com on 1/18/16.
+ */
+
+public class EmployeeDao {
+  private static final Logger LOGGER = Logger.getLogger(Employee.class.getName());
+
+  private static final String EMPLOYEE_INSERT = "INSERT INTO employee (name, address, designation, phone) VALUES (?,?,?,?)";
+  private static final String EMPLOYEE_SELECT_ALL = "SELECT * FROM employee order by id limit ? offset ?";
+  private static final String EMPLOYEE_SELECT_BY_ID = "SELECT * FROM employee where id = ?";
+  private static final String EMPLOYEE_UPDATE_BY_ID = "update employee set name = ?, address = ?, designation = ?, phone = ? where id = ?";
+  private static final String EMPLOYEE_DELETE_BY_ID = "delete from employee where id = ?";
+  private static final String EMPLOYEE_GET_COUNT = "select count(*) as total from employee";
+
+  public void create(Employee employee) throws DataException {
+    try(Connection conn = DBConnection.getConnection();
+      PreparedStatement statement = conn.prepareStatement(EMPLOYEE_INSERT)
+    ){
+      statement.setString(1, employee.getName());
+      statement.setString(2, employee.getAddress());
+      statement.setString(3, employee.getDesignation());
+      statement.setString(4, employee.getPhone());
+      statement.executeUpdate();
+      ResultSet resultSet = statement.getGeneratedKeys();
```

Comment:
Can't we also initialize ResultSet inside try. Also if you are initializing you have to close the resources.